src/nodes/CustomErrorDefinition.js

```diff
@@ -0,0 +1,19 @@
+const {
+  doc: {
+    builders: { concat }
+  }
+} = require('prettier/standalone');
+
+const printSeparatedList = require('./print-separated-list');
+
+const parameters = (node, path, print) =>
+  node.parameters && node.parameters.length > 0
+    ? printSeparatedList(path.map(print, 'parameters'))
+    : '';
+
+const CustomErrorDefinition = {
+  print: ({ node, path, print }) =>
+    concat(['error ', node.name, '(', parameters(node, path, print), ')', ';'])
```

Comment:
the last `')', ';'` could just be `');'`

Reply:
Oops, good catch, that happened because I copied and modified the `EventDefinition` printer :sweat_smile: